cloud/shared/bin/doctor_impl

```diff
@@ -133,7 +130,7 @@ class AwsLoginCheck(Check):
         return "AWS login"
 
     def should_skip(self):
-        return self._get_cloud_provider() not in ['aws', 'azure']
+        return self._get_cloud_provider() != 'aws'
```

Comment:
issue (blocking): the AWS CLI is needed for Azure deployments as well since our only azure deployment atm uses AWS SES

Reply:
Ah good to know! Reverted, and added a comment to clarify.